libtenzir/src/multi_series_builder_argument_parser.cpp

```diff
@@ -80,6 +80,8 @@ auto multi_series_builder_argument_parser::add_settings_to_parser(
   if (add_unflatten_option) {
     parser.add("unflatten", unnest_);
   }
+  parser.add("_timeout", timeout_);
```

Comment:
Mind calling this `_batch_timeout` to avoid conflicts with other parsers or sources that actually have a timeout option?